docs/architecture/event-flows-proposal.md

```diff
@@ -0,0 +1,77 @@
+# Eventing flow proposal
+
+## Overview
+
+This document shows a proposal of two ways of eventing flow we want to support with use of the [Management Plane](./terminology.md#management-plane)
+
+## Eventing with Kyma internal Event Bus
+
+![Management Plane Components](assets/internal-event-flow.svg)
```

Comment:
please update a diagram with not EventService, but EventGateway and say it supports:
- http
- mqtt
- others
then also many types of auth, like oauth2 (authorization) + openID Connect (authentication)